chpl/chpl-api/e2e/script/newman-run-dev.js

```diff
@@ -0,0 +1,76 @@
+const path= require('path'),
+async= require('async'),
+newman= require('newman');
+
+const collection_path= path.resolve('chpl/chpl-api/e2e/collections')
+const reports_path= path.resolve('newman')
+
+const commonOptions = {
+	reporters: ['junit','cli'],
+	insecure: true,
+	environment : 'chpl/chpl-api/e2e/env/dev.postman_environment.json'
+}
+
+const acbControllerTests={
+	...commonOptions,
+	collection: collection_path + '/acb-controller.postman_collection.json',
+	reporter : {
+		junit :{
+			export: reports_path + '/acb-controller-tests.xml'
+		}
+	}
+}
```

Comment:
It would be good to decide on a consistent formatting and stick to it. This can be done automatically with a scripted formatter if desired. Above we have examples of:
type name= value
type name = {
type name={
type : {
type :{

I would suggest updating to the following options consistently to match our existing formatting:
type name = value
type : {

If you have a conviction for another formatting, or keeping the varying formatting, I won't hold back the PR. Just mentioning as, even though not all that important, this would not get through non-testing code PR review. The pros are that It helps readability, but, more importantly if all style is the same from all users it helps identify actual impactful changes vs style changes in new PRs.

Also, it is best practice to have a new line at the end of file. Only mentioning cause I have already made this comment...it's not a big deal enough to hold back a PR.

Reply:
Agree with Dan's comments, except for `type : {`. In the UI code those are always `type: {`

I'd actually suggest adding eslint with the same configs as the UI project to this one, targeted at the js files, so we can be consistent across the projects while in the same language. I don't know if we have a json linter anywhere, but that would be good to investigate as well

Reply:
Good point Andrew - I guess I should have linted my comment :-)